functions/src/modules/SQR.functions.ts

```diff
@@ -278,16 +312,39 @@ export const processSubmission = functions.database
   if (allSubmissionsSnapshot.exists()) {
     const allSubmissions = allSubmissionsSnapshot.val();
 
+    const allotments = (await db
+        .ref(`/files/${list}`)
+        .orderByChild("soundQualityReporting/assignee/emailAddress")
+        .equalTo(submission.author.emailAddress).once("value")
+    ).val();
+
+    const currentSet = [];
+
+    Object.entries(allotments).forEach(
+        ([fileName, { languages, soundQualityReporting: { timestampGiven, status } }]: any) => {
+          currentSet.push({
+            fileName,
+            status,
+            timestampGiven: timestampGiven ? moment(timestampGiven).format("M/D/YYYY") : "",
```

Comment:
Are you sure this is not possible to format in the template itself? Sorry if I was asking this already.

Reply:
I was unable to format with the template.